lua/terrortown/lang/de.lua

```diff
@@ -1595,307 +1594,305 @@ L.label_roles_credits_award_repeat = "Mehrfache Belohnung"
 L.label_roles_newroles_enabled = "Aktiviere eigene Rollen"
 L.label_roles_max_roles = "Obere Grenze für Rollen"
 L.label_roles_max_roles_pct = "Prozentuale obere Grenze für Rollen"
---L.label_roles_max_baseroles = "Upper base role limit"
---L.label_roles_max_baseroles_pct = "Upper base role limit by percentage"
---L.label_detective_hats = "Enable hats for policing roles like the Detective (if player model allows to have them)"
+L.label_roles_max_baseroles = "Obere Basisrollenbegrenzung"
+L.label_roles_max_baseroles_pct = "Obere Basisrollenbegrenzung in Prozent"
+L.label_detective_hats = "Aktiviere Hüte für Polizeirollen wie den Detektiv (sofern das Spielermodell dies zulässt)"
 
---L.ttt2_desc_innocent = "An Innocent has no special abilities. They have to find the evil ones among the terrorists and kill them. But they have to be careful not to kill their teammates."
---L.ttt2_desc_traitor = "The Traitor is the enemy of the Innocent. They have an equipment menu with which they are being able to buy special equipment. They have to kill everyone but their teammates."
+L.ttt2_desc_innocent = "Ein Unschuldiger hat keine besonderen Fähigkeiten. Sie müssen die bösen unter den Terroristen finden und sie töten. Dabei müssen sie jedoch vorsichtig sein, ihre Teammitglieder nicht zu töten."
+L.ttt2_desc_traitor = "Der Verräter ist der Feind der Unschuldigen. Sie verfügen über ein Ausrüstungsmenü, mit dem sie spezielle Ausrüstung kaufen können. Ihr Ziel ist es, jeden zu töten, außer ihre Teammitglieder."
 L.ttt2_desc_detective = "Der Detektiv ist derjenige, dem die Unschuldigen trauen können. Aber wer sind die Unschuldigen? Der Detektiv muss genau das herausfinden. Ihr Ausrüstungsshop hilft ihnen vielleicht dabei."
 
 -- 2021-10-10
 L.button_reset_models = "Spielermodelle Zurücksetzen"
 
 -- 2021-10-13
 L.help_roles_credits_award_kill = "Ein weiterer Weg Ausrüstungspunkte zu erhalten ist es wichtige Spieler mit 'offenen Rollen' (wie beispielsweise dem Detektiv) zu töten. Wenn die Rolle des Mörders dies aktiviert hat, dann bekommt der Spieler die hier definierte Anzahl an Ausrüstungspunkten."
---L.help_roles_credits_award = [[
---There are two different ways to be awarded credits in base TTT2:
+L.help_roles_credits_award = [[
+Es gibt zwei verschiedene Möglichkeiten, um in Basis-TTT2 Credits zu erhalten:
 
---1. If a certain percentage of the enemy team is dead, the whole team is awarded credits.
---2. If a player killed a high value player with a 'public role' such as a Detective, the killer is awarded credits.
+1. Wenn ein bestimmter Prozentsatz des feindlichen Teams tot ist, erhaltet das gesamte Team Credits.
+2. Wenn ein Spieler einen Spieler mit einer 'öffentlichen Rolle' wie einem Detektiv getötet hat, werden dem Mörder Credits verliehen.
```

Comment:
```suggestion
2. Wenn ein Spieler einen Spieler mit einer 'bekannten Rolle' wie einem Detektiv getötet hat, werden dem Mörder Credits verliehen.
```
Selbes wie oben. Wobei ich dein öffentlich in dem Kontext klarer finde. Sollten wir besprechen